draft-ietf-mls-protocol.md

```diff
@@ -1287,10 +1287,13 @@ state as follows:
   the null optional value
 * Update the ratchet tree by replacing nodes in the direct
   path from the removed leaf using the information in the Remove message
-* Reduce the size of the roster and the tree until the rightmost
-  element roster element and leaf node are non-null
 * Update the ratchet tree by setting to blank all nodes in the
-  direct path of the removed leaf
+  direct path of the removed leaf, and also setting the root node
+  to blank
```

Comment:
This is redundant, given that the root path is on the direct path of every node.  Unless there's some definition that comes up one short?

Reply:
From section 5.1: "The direct path of a root is the empty list, and of any other node is the concatenation of that node with the direct path of its parent."

Direct paths do not contain the root node.